src/basho_bench_stats_writer_csv.erl

```diff
@@ -204,7 +204,8 @@ stringify_stats(_RunStatsType=csv, RunMetrics) ->
             [StrVal|Acc]
         end, [], UniqueHeaders),
 
-        [io_lib:format("~s\n", [string:join([atom_to_list(Op)|CsvLine],", ")])|Lines]
+        [io_lib:format("~s\n", 
+            [string:join( [atom_to_list(Op) | CsvLine], ", ")]) | Lines]
```

Comment:
This line is very confusing. It takes a fair bit of parsing to understand that it's appending into a list as opposed to a list comprehension or something else. I would do something along the lines of one of the following:

```
OpStr = string:join( [atom_to_list(Op) | CsvLine], ", "),
[io_lib:format("~s\n", [OpStr]) | Lines]
```

or

```
[
    io_lib:format("~s\n", [
        string:join( [atom_to_list(Op) | CsvLine], ", ")])
    ])
    | Lines
]
```